### ISSUE TITLE
[Community] Member Profile: Sandra Ashipala

### ISSUE THREAD

**Issue by saurabh100ni** [layer5io/layer5#5027]
**Current Behavior**

@sandramsc has been a consistent contributor and community member.

**Desired Situation**
Let's recognize @sandramsc as a contributor and community member by creating a profile on https://layer5.io/community/members.

- GitHub: @sandramsc
- Twitter: [sandraamsc](https://twitter.com/sandraamsc)
- LinkedIn: https://www.linkedin.com/in/sandraashipala/
- Meshery Cloud: 
- Link to profile picture:

A detailed explanation of how to set up a community member profile can be found in the [CONTRIBUTING.md](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---

**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](https://slack.layer5.io) for access.

**Comment by saurabh100ni** [comment]
@sandramsc can you share following :
- Meshery Cloud:
- Link to profile picture:

**Comment by saurabh100ni** [comment]
Go on @VivekBaldha0712 😎🚀

**Comment by VivekBaldha0712** [comment]
Hey @sandramsc, Will you please provide your bio

**Comment by sandramsc** [comment]
> Hey @sandramsc, Will you please provide your bio

**Here it is:**

With a diverse international background, I've collaborated with companies in Germany, France, and Italy, gaining a broad perspective on software development. Thriving in diverse teams, I've worked with individuals from various backgrounds and cultures. These international collaborations expanded my professional horizons and instilled a deep appreciation for diversity's power in driving innovation.

I hold a Master's in Computer Science from the Technical University of Berlin. Currently, I'm enhancing my skills in DevOps and Python.  My journey of growth includes active participation in the Layer5 open-source community, where I eagerly contribute and thrive in the world of service mesh technology.
